packages/vm/src/calls.rs

```diff
@@ -232,6 +232,35 @@ mod tests {
             .unwrap();
     }
 
+    #[test]
+    fn call_migrate_works() {
+        let mut instance = mock_instance(&CONTRACT, &[]);
+
+        // init
+        let info = mock_info("creator", &coins(1000, "earth"));
+        let msg = r#"{"verifier": "verifies", "beneficiary": "benefits"}"#.as_bytes();
```

Comment:
```suggestion
        let msg = br#"{"verifier": "verifies", "beneficiary": "benefits"}"#;
```

I know this is copied from old code. But we should really change that (https://github.com/CosmWasm/cosmwasm/issues/732).